crates/server/src/server/server_future.rs

```diff
@@ -868,22 +868,40 @@ impl<R: ResponseHandler> ResponseHandler for ReportingResponseHandler<R> {
         let additional_count = response_info.additional_count();
         let response_code = response_info.response_code();
 
-        info!("request:{id} src:{proto}://{addr}#{port} {op}:{query}:{qtype}:{class} qflags:{qflags} response:{code:?} rr:{answers}/{authorities}/{additionals} rflags:{rflags}",
-            id = rid,
-            proto = self.protocol,
-            addr = self.src_addr.ip(),
-            port = self.src_addr.port(),
-            op = self.request_header.op_code(),
-            query = self.query.name(),
-            qtype = self.query.query_type(),
-            class = self.query.query_class(),
-            qflags = self.request_header.flags(),
-            code = response_code,
-            answers = answer_count,
-            authorities = authority_count,
-            additionals = additional_count,
-            rflags = rflags
-        );
+        if self.queries.len() == 1 {
```

Comment:
Should we consider instead emitting one event for the request and a separate event for the query?